model/user.go

```diff
@@ -47,6 +47,77 @@ const (
 	USER_PASSWORD_MAX_LENGTH  = 72
 )
 
+// This is a struct for a user interaction
+// A user interaction will represent one row in the results of the
+// GetUserInteractions query in store/sqlstore/user_store.go
+// **Note: Using pointers as property types allows for null values
+type UserInteraction struct {
+	// The type of interaction
+	// enum('Reaction', 'Mention', 'Reply', 'DirectMessage')
+	Interaction_Type *string `json:"Interaction_Type"`
+
+	// The username of the user that this interaction took place with
+	Username *string `json:"Username"`
+
+	// Denotes whether or not the current user was the recipient of this interaction
+	Is_Recipient *bool `json:"Is_Recipient"`
+
+	// The context in which this interaction occured (either 'Course' or the learning group type's prefix (ex. 'plg'))
+	Context *string `json:"Context"`
+
+	// The type of channel that this interaction took place in
+	// enum('P', 'O', 'D', 'G')
+	//     P = private channel
+	//     O = public channel
+	//     D = direct message (1 on 1)
+	//     G = group direct message
+	Channel_Type *string `json:"Channel_Type"`
```

Comment:
The comment is is super helpful! Anything I might have thought about this Channel_Type fields contents was not what it is, but having read the comment, I can easily carry that knowledge around when looking at other code using it.